frontend/src/components/VTag/VTag.vue

```diff
@@ -4,9 +4,10 @@
     size="small"
     variant="filled-gray"
     class="label-bold"
-    :href="href"
-    >{{ title }}</VButton
-  >
+    v-bind="$props"
+    v-on="$listeners"
```

Comment:
Why were these added?

Reply:
Normally, these should just pass through, but I think because we use 2 levels of components here: `VButton` that is `as` `VLink`, they are not being passed without these.